README.md

```diff
@@ -99,15 +99,13 @@ follow these steps:
 > rm -rf ~/.cabal
 > ```
 >
-> You may prefer a less invasive approach if you have custom global
-> Cabal configurations:
+> A less invasive approach is to point Cabal to a fresh store folder, 
+> but in this case you won't see the `Downloading` phase in the profiling 
+> output:
 >
 > ```shell
-> rm -rf ~/.cabal/store
+> cabal --store-dir=$(mktemp -d) build all
```

Comment:
Just for my understanding, this is because with `--store-dir`, cabal reuses downloaded sources from `~/.cabal` and that's why we don't see the Downloading phase?

Reply:
Yes, `--store-dir` overrides only `~/.cabal/store`, but other contents of `~/.cabal` such as `~/.cabal/packages` remains in use.